helpers/helper-translation-discussion.php

```diff
@@ -300,19 +312,69 @@ public static function get_original_from_post_id( int $post_id ) {
 		return $terms[0]->slug;
 	}
 
+	public static function is_temporary_post_id( $post_id ) {
```

Comment:
```suggestion
	/**
	 * Indicates whether the post is temporal or not.
	 *
	 * True if is temporal.
	 * False if is not temporal.
	 *
	 * @since 0.0.2
	 *
	 * @param int|string $post_id The post ID.
	 *
	 * @return bool
	 */
	public static function is_temporary_post_id( $post_id ) {
```